diff -up baloo-5.54.0/src/file/fileindexerconfig.cpp.omv~ baloo-5.54.0/src/file/fileindexerconfig.cpp
--- baloo-5.54.0/src/file/fileindexerconfig.cpp.omv~	2019-01-07 05:14:17.909517286 +0100
+++ baloo-5.54.0/src/file/fileindexerconfig.cpp	2019-01-07 05:14:26.652619865 +0100
@@ -389,7 +389,7 @@ void FileIndexerConfig::setDatabaseVersi
 
 bool FileIndexerConfig::indexingEnabled() const
 {
-    return m_config.group("Basic Settings").readEntry("Indexing-Enabled", true);
+    return m_config.group("Basic Settings").readEntry("Indexing-Enabled", false);
 }
 
 uint FileIndexerConfig::maxUncomittedFiles() const
