diff -up baloo-5.79.0/src/lib/Baloo.pc.cmake.2~ baloo-5.79.0/src/lib/Baloo.pc.cmake
--- baloo-5.79.0/src/lib/Baloo.pc.cmake.2~	2021-02-13 18:24:59.571770387 +0100
+++ baloo-5.79.0/src/lib/Baloo.pc.cmake	2021-02-14 00:02:45.888596113 +0100
@@ -8,5 +8,5 @@ Description: Baloo is a file indexing an
 URL: https://www.kde.org
 Requires: Qt5Core
 Version: ${BALOO_VERSION_STRING}
-Libs: -L${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_LIBDIR} -lKF5Baloo
-Cflags: -I${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_INCLUDEDIR_KF5}/Baloo
+Libs: -lKF5Baloo
+Cflags: -I${KDE_INSTALL_INCLUDEDIR_KF5}/Baloo
